@shared/api/helpers/getIconUrlFromIssuer.ts

```diff
@@ -43,6 +43,7 @@ export const getIconUrlFromIssuer = async ({
   try {
     /* First, check our localStorage cache in Background to see if we've found this url before */
     ({ iconUrl } = await sendMessageToBackground({
+      activePublicKey: null,
```

Comment:
same question here, I guess it also doesn't matter for fetching the icon

Reply:
Yup, same. Fetching an icon is not something specific to an account